src/main/java/com/faforever/client/chat/ChatUserItemController.java

```diff
@@ -292,10 +355,26 @@ public void onMouseEnterUsername() {
     if (player == null || player.getChatOnly() || usernameLabel.getTooltip() != null) {
       return;
     }
-
     Tooltip tooltip = new Tooltip();
     usernameLabel.setTooltip(tooltip);
-    clanLabel.setTooltip(tooltip);
+
+    Runnable setClanTooltip = new Runnable() {
+      @Override
+      public void run() {
+        if (clanMenu.getTooltip() == null && clan != null) {
+          Tooltip clanTooltip = new Tooltip();
+          clanMenu.setTooltip(clanTooltip);
+          if (clan.getDescription() == null) {
+            clan.setDescription("-");
+          }
+          clanTooltip.setText(i18n.get("clan.clanName") + "\n" + clan.getClanName() + "\n\r" + i18n.get("clan.describtion") + "\n" + clan.getDescription() + "\n\r" + i18n.get("clan.clanMembers") + "\n" + clan.getClanMembers() + "\n\r" + i18n.get("clan.leader") + "\n" + clan.getLeaderName());
+        }
+      }
+    };
+
+    executorService.submit(setClanTooltip);
+    executorService.shutdown();
+
 
     tooltip.textProperty().bind(Bindings.createStringBinding(
         () -> i18n.get("userInfo.ratingFormat", getGlobalRating(player), getLeaderboardRating(player)),
```

Comment:
This could be moved up to the setting of the location of the creation of the usernameLabel tooltip, now it's seperated by the clan tooltip.